virtual_rainforest/models/hydrology/hydrology_model.py

```diff
@@ -338,19 +393,17 @@ def update(self, time_index: int) -> None:
 
         # Calculate total soil moisture (before rainfall) in mm
         # To find out how much rain can be taken up by the soil before rain goes to
-        # runoff, the relative water content (between 0 and 1) is converted to mm with
-        # this equation:
+        # runoff, the volumetric relative water content (between 0 and 1) is converted
+        # to mm with this equation:
         # water content in mm = relative water content / 100 * depth in mm
         # Example: for 20% water at 40 cm this would be: 20/100 * 400mm = 80 mm
         # TODO We treat the soil as one bucket, in the future, there should be a
         # flow between layers and a gradient of soil moisture and soil water potential
         total_soil_moisture_mm = (
-            self.data["soil_moisture"]
+            self.data["soil_moisture"].isel(layers=-1)
             * (-self.constants.meters_to_millimeters)
-            * self.data["layer_heights"]
-            .where(self.data["layer_heights"].layer_roles == "soil")
-            .dropna(dim="layers")
-        ).sum(dim="layers")
+            * self.data["layer_heights"].isel(layers=-1)
```

Comment:
Is this simplification possible because you are basically treating all soil layers as a single "bucket"?

Reply:
yes, that's what it is.

Reply:
In the previous version, I made an error when summing over all depths, I treated the layer depths like layer thickness. If I want one bucket I just need the depth of the lowest layer.